Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v4.0.0-beta.3
+
+- [fixed] Fixed symbol collisions with the `Crashlytics` SDK and added a warning not to include both (#4753, #4755)
```

Comment:
Let's change "`Crashlytics` SDK" to "Firebase Crashlytics SDK" (without code font)

Reply:
This is referring to the old Crashlytics SDK - how should we refer to that?

Reply:
maybe "Fabric Crashlytics SDK"?

Reply:
Oh oops, just read this thread. Let's change this to "...with the legacy Fabric SDK".